app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:weightSum="1"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <LinearLayout
```

Comment:
if i put it in a frame layout it puts them both ontop of each other and if i gave them a frame layout each then it is just more code